ConsoleApp/Program.cs

```diff
@@ -0,0 +1,86 @@
+﻿byte variableByte = 12;
+short variableShort = 255;
+int variableInt = 5045;
+long variableLong = 4890238;
+bool variableBool = true;
+char variableChar = '!';
+float variableFloat = 0.1f;
+double variableDouble = 0.345;
+decimal variableDecimal = 0.64578697m;
+string variableString = "Hi";
+
+Console.WriteLine("Division: ");
+decimal div = variableDecimal / variableInt;
+Console.WriteLine(div);
+Console.WriteLine();
+
+Console.WriteLine("Multiplication: ");
+double mult = variableDouble * variableFloat;
+Console.WriteLine(mult);
+Console.WriteLine();
+
+Console.WriteLine("Concatenation: ");
+string conc = variableString + variableChar;
+Console.WriteLine(conc);
+Console.WriteLine();
+
+Console.WriteLine("Exclusive OR: ");
+int exOR = variableShort ^ variableByte;
+Console.WriteLine(exOR);
+Console.WriteLine();
+
+Console.WriteLine("Even");
+bool even = variableLong % 2 == 0 ? variableBool : false;
+Console.WriteLine(even);
+Console.WriteLine();
+
+Console.WriteLine("Negation: ");
+Console.WriteLine(!variableBool);
+Console.WriteLine();
+
+
+
+Console.WriteLine("Enter the number for x: ");
+int x;
+int.TryParse(Console.ReadLine(), out x);
+
+//operation #1
+double resOfOperation_1 = -6 * x ^ 3 + 5 * x ^ 2 - 10 * +15;
```

Comment:
x ^ 3 is not the same as Math.Pow(x, 3) or x * x * x, it won't do what you expecting :)

Reply:
I know it. I did it accidentally, as I understand it will do "exclusive or" operation.

Reply:
`= -6 * x ^ 3 + 5 * x ^ 2 - 10 * +15;` - and I did so in begin.